android/java/org/chromium/chrome/browser/rewards/BraveRewardsCreatorPanelFragment.java

```diff
@@ -294,16 +294,21 @@ private void setPublisherNoteText(int pubStatus, int walletStatus) {
                 notePart1 = getResources().getString(
                         R.string.brave_ui_site_banner_unverified_notice_text);
                 Log.d(TAG, "User is verified and publisher is unverified");
-            } else if ((pubStatus == PublisherStatus.UPHOLD_VERIFIED
-                               && !walletType.equals(BraveWalletProvider.UPHOLD))
-                    || (pubStatus == PublisherStatus.BITFLYER_VERIFIED
-                            && !walletType.equals(BraveWalletProvider.BITFLYER))
-                    || (pubStatus == PublisherStatus.GEMINI_VERIFIED
-                            && !walletType.equals(BraveWalletProvider.GEMINI))) {
-                notePart1 = getResources().getString(
-                        R.string.brave_ui_site_banner_different_verified_notice_text);
-                Log.d(TAG,
-                        "User is verified and publisher is verified but not with the same provider");
+            } else if (pubStatus == PublisherStatus.UPHOLD_VERIFIED
+                    && !walletType.equals(BraveWalletProvider.UPHOLD)) {
+                notePart1 = String.format(
+                        getString(R.string.brave_ui_site_banner_different_verified_notice_text1),
+                        walletType, BraveWalletProvider.UPHOLD);
```

Comment:
it seems like we have the same code for different else if parts. is there anything i am missing ?

Reply:
Yes code is almost same except last parameter. like BraveWalletProvider.UPHOLD
R.string.brave_ui_site_banner_different_verified_notice_text1 this requires two parameters.
%1$s, %2$s
i.e.
Which custodian user is using and which custodian creator is verified.

Reply:
we can probably create a private function to get custodian name. that way we can avoid duplicated code.

Reply:
updated